### PR TITLE
Fix conformance tests in oneDAL public CI

### PR DESCRIPTION
**Pull request by OnlyDeniko**

### CODE REVIEW

**Comment by OnlyDeniko** [comment]
[public CI in oneDAL with this branch](https://dev.azure.com/daal/DAAL/_build/results?buildId=15570&view=logs&j=6bc891dc-3c29-5347-0ca2-82137807ca2b&t=9df9471f-ff24-5297-2bec-ab9877350b47)

**Comment by agorshk** [comment]
Ok, from my side, please wait to Michael review

File: scripts/CMakeLists.txt [hunk 1]
```diff
@@ -79,7 +79,7 @@ if(IFACE STREQUAL "host")
     if(WIN32)
         list(APPEND ONEDAL_LIBRARIES "onedal_dll")
     else()
-        list(APPEND ONEDAL_LIBRARIES "onedal")
+        set(ONEDAL_LIBRARIES "onedal" "onedal_core" "onedal_thread")
```
Comment (michael-smirnov) [inline scripts/CMakeLists.txt:82]: Please, make it consistent for Windows:
I suggest to remove lines 29-35, and place the same lines in Windows branches (80, 91).
For example,
`set(ONEDAL_LIBRARIES "onedal_dll", "onedal_core_dll", "onedal_thread_dll")`

**Review by michael-smirnov — APPROVED** [review]
Assume CI is green

File: scripts/CMakeLists.txt [hunk 2]
```diff
@@ -77,9 +69,9 @@ if(IFACE STREQUAL "host")
     set(TARGET "_onedal_py_host")
 
     if(WIN32)
-        list(APPEND ONEDAL_LIBRARIES "onedal_dll")
+        set(ONEDAL_LIBRARIES "onedal_dll", "onedal_core_dll", "onedal_thread_dll")
```
Comment (michael-smirnov) [inline scripts/CMakeLists.txt:72]:
```suggestion
        set(ONEDAL_LIBRARIES "onedal_dll" "onedal_core_dll" "onedal_thread_dll")

```
```suggestion
        set(ONEDAL_LIBRARIES "onedal_dll" "onedal_core_dll" "onedal_thread_dll")
```

File: scripts/CMakeLists.txt [hunk 3]
```diff
@@ -88,9 +80,9 @@ elseif(IFACE STREQUAL "dpc")
     set(TARGET "_onedal_py_dpc")
 
     if(WIN32)
-        list(APPEND ONEDAL_LIBRARIES "onedal_dpc_dll")
+        set(ONEDAL_LIBRARIES "onedal_dll", "onedal_core_dll", "onedal_thread_dll")
```
Comment (michael-smirnov) [inline scripts/CMakeLists.txt:83]:
```suggestion
        set(ONEDAL_LIBRARIES "onedal_dpc_dll" "onedal_core_dll" "onedal_thread_dll")

```

**Review by michael-smirnov — CHANGES_REQUESTED** [review]

**Review by michael-smirnov — APPROVED** [review]

**Pull request merged**